src/utils/Tmdb.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * TMDB API client configuration
+ * @type {{key: string, baseUrl: string, imageBaseUrl: string}}
+ */
+const apiConfig = {
+  key: '5265606bb69e99437c85eb04dc6f29b5',
+  baseUrl: 'https://api.themoviedb.org/3/',
+  imageBaseUrl: 'http://image.tmdb.org/t/p/',
+};
+
+/**
+ * An abstraction layer above TMDB API https://developers.themoviedb.org/3/getting-started/introduction
+ */
+class _Tmdb {
+  constructor() {
+    this.apiConfig = apiConfig;
+    this.imageConfig = undefined;
+  }
+
+  /**
+   * Discover movies as per https://developers.themoviedb.org/3/discover/movie-discover
+   * @param {Object} searchParams - see query() for details
+   * @returns {Promise<Response>} - see query() for details
```

Comment:
I am not sure that we need such detailed comments here because you can replace typing with the flow.

But if you want - you can leave it. But anyway, pls add types 😺